pkg/driver/controller.go

```diff
@@ -124,6 +126,20 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		Encrypted:        isEncrypted,
 		KmsKeyID:         kmsKeyId,
 	}
+
+	// Shall we restore a snapshot?
```

Comment:
Do we still need this comment? If the snapshot ID is provide, the create volume API call will create the volume from snapshot, right?

Reply:
Yes. Will remove it with reabse.